pootle/core/search/__init__.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
```

Comment:
@dwaynebailey I think it might be better to name the module `pootle.core.tm` instead.